bindings/go/main_test.go

```diff
@@ -36,7 +36,7 @@ func getRandFieldElement(seed int64) Bytes32 {
 	// This leaves the last byte in fieldElementBytes as
 	// zero, which guarantees it's a canonical field element.
 	var fieldElementBytes Bytes32
-	copy(fieldElementBytes[:], bytes)
+	copy(fieldElementBytes[1:], bytes)
```

Comment:
Hmm, what is this change again?

Reply:
So a little above this we generate 31 random bytes. Previously, this would copy 31 bytes to the 32-byte field element and leave the last byte as zero... so we know it's valid. Now, since it's big-endian, the first byte should be zero. This changes the starting position of the copy so the first byte remains zero. If you want, I can add a comment or do it differently. For example, generate/copy 32 bytes and manually set the first one to zero.

Reply:
Ah that makes sense. I guess then we need https://github.com/ethereum/c-kzg-4844/pull/305/files#r1198905335 ?

Reply:
Ah yes, thank you. Fixed.